python/python/bystro/covariance/tests/test_base_covariance.py

```diff
@@ -1,7 +1,22 @@
-import numpy as np
-import numpy.linalg as la
+import numpy as np  # type: ignore
+import numpy.linalg as la  # type: ignore
 import scipy.stats as st  # type: ignore
-from bystro.covariance._base_covariance import BaseCovariance, _score_samples
+from _base_covariance import (
```

Comment:
This import was wrong, needed a leading period or to use the absolute import, which we prefer